src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/config/HmppsExternalUsersApiExceptionHandler.kt

```diff
@@ -102,6 +104,14 @@ class HmppsExternalUsersApiExceptionHandler {
       )
   }
 
```

Comment:
Question - could we have a generic not found exception rather than one per domain type? Might be one for a separate ticket

Reply:
I think that is a good idea